src/main/resources/application.properties

```diff
@@ -60,6 +60,8 @@ email.subject: Email Subscription Notification
 notification.failAttempt: 3
 notification.ttl.value: 600
 
+ldap.enabled: false
+
```

Comment:
You have this variable already set on line 74. So remove this one and keep all ldap parameters grouped together in the properties file.

Reply:
Done